### ISSUE TITLE
No matching distribution found for capi-param-builder>=0.1.0-dev0

### ISSUE THREAD

**Issue by tiagoskaneta** [facebook/facebook-python-business-sdk#684]
I'm having an issue installing the latest (21.0.0) version:
```
Collecting facebook_business==21.0.0
...
ERROR: Could not find a version that satisfies the requirement capi-param-builder>=0.1.0-dev0 (from facebook-business) (from versions: none)
ERROR: No matching distribution found for capi-param-builder>=0.1.0-dev0
```

Does't look like `capi-param-builder`  is avaiable in pypi.

**Comment by liliarizona** [comment]
@tiagoskaneta Thx for reporting the issue. We have released version 21.0.1 that fixed the issue. Could you try installing again?

**Comment by tiagoskaneta** [comment]
@liliarizona thanks, that has indeed fixed the issue.
